### PR TITLE
[run] Fix ansible-runner artifacts cleanup

### LINKED ISSUE #92: runner leaves residue in playbook directory
the runner artifacts should be located within the EE if running with an EE or possibly a tmp directory when running without

### PR DESCRIPTION
**Pull request by ganeshrn**
Depends-On: https://github.com/ansible/ansible-navigator/pull/288

Fixes https://github.com/ansible/ansible-navigator/issues/255
Fixes #92 

* use temp dir and runner artifact handler/callback
*  Update tmux_session logic to combine the command executed value
   shown within the pane as longer commands can be split into
   mulitple lines when capturing the pane
*  Add integration test cases from run action

### CODE REVIEW

File: tests/integration/actions/run/base.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+from typing import Optional
+from ....defaults import FIXTURES_COLLECTION_DIR
+from ....defaults import FIXTURES_DIR
+from ..._common import fixture_path_from_request
+from ..._common import update_fixtures
+from ..._tmux_session import TmuxSession
+
+
+# run playbook
+run_fixture_dir = os.path.join(f"{FIXTURES_DIR}", "integration", "actions", "run")
```
Comment (relrod) [inline tests/integration/actions/run/base.py:17]:
```suggestion
run_fixture_dir = os.path.join(FIXTURES_DIR, "integration", "actions", "run")
```

**Comment by cidrblock** [comment]
multiple instances of ansible-navigator will remove each others artifact directories with rotate=1, 

Let do this. /tmp/ansible-navigator/uuid/

where uuid is generated when the navigator starts

and we can delete the directory before we exit, we cna store the uuid in the config subsystem internals

File: tests/integration/actions/run/base.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                received_output[idx] = mask
+            else:
+                for out in ["duration:", "playbook:", "start:", "end:", "task_path:"]:
+                    if out in line:
+                        received_output[idx] = mask
+
+            if self.UPDATE_FIXTURES:
+                update_fixtures(request, index, received_output, comment)
+            dir_path, file_name = fixture_path_from_request(request, index)
+            with open(f"{dir_path}/{file_name}") as infile:
```
Comment (relrod) [inline tests/integration/actions/run/base.py:69]:
```suggestion
            with open(os.path.join(dir_path, file_name)) as infile:
```

**Review by cidrblock — APPROVED** [review]
are the artifact file intentional?

File: tests/fixtures/integration/actions/run/site-artifact-2021-05-17T23:14:28.831238+00:00.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (cidrblock) [inline tests/fixtures/integration/actions/run/site-artifact-2021-05-17T23:14:28.831238+00:00.json:1]: here

File: tests/fixtures/integration/actions/run/site-artifact-2021-05-17T23:14:28.831238+00:00.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+    "version": "1.0.0",
```
Comment (cidrblock) [inline tests/fixtures/integration/actions/run/site-artifact-2021-05-17T23:14:28.831238+00:00.json:2]: here

**Comment by cidrblock** [comment]
recheck

**Comment by cidrblock** [comment]
recheck

**Comment by cidrblock** [comment]
Recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by cidrblock** [comment]
recheck

**Pull request merged**